digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -1529,4 +1529,11 @@ object RoadAddressDAO {
           valid_from <= sysdate and floating = 0 and ely = $ely"""
     queryList(query)
   }
+
+  def getLinkCPValue(roadAddressId: Long): Long = {
+    val query = s"""SELECT ra.calibration_points
+                    FROM road_address ra
+                    WHERE ra.id=$roadAddressId AND ROWNUM=1"""
```

Comment:
road_address table's id-field is already unique, so " AND ROWNUM=1" can be removed.